apps/angular/src/app/features/anime-dashboard/components/anime-table/anime-table.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatTableModule } from '@angular/material/table';
+import { EmptyPipe } from '@js-camp/angular/shared/pipes/empty.pipe';
+import { BasicProgressSpinnerComponent } from '@js-camp/angular/shared/components/basic-progress-spinner/basic-progress-spinner.component';
+import { Anime } from '@js-camp/core/models/anime';
+
+/** Table of anime list component. */
+@Component({
+	selector: 'camp-anime-table',
+	standalone: true,
+	templateUrl: './anime-table.component.html',
+	styleUrl: './anime-table.component.css',
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	imports: [CommonModule, MatTableModule, EmptyPipe, BasicProgressSpinnerComponent],
+})
+export class AnimeTableComponent {
+	/** Anime list from parent component. */
```

Comment:
Too much context. It is obvious that this anime list comes from parent because you have `@Input`

```suggestion
	/** Anime list. */
```